tests/cms/test_firebase_client_api.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from integreat_cms.api_client_firebase.push_notification_sender import PushNotificationSender
+from integreat_cms.cms.models.push_notifications.push_notification import PushNotification
+from django.core.exceptions import ImproperlyConfigured
+
+#######################################################################
+# Test for api_client_firebase / push_notification_sender-class (pns) 
+# @TODO
+# def test_pns_send_pn()
+# def test_pns_send_all()
+#######################################################################
+class TestApiClientFirebase:
```

Comment:
What's the plan with the tests, do you want to include basic tests in this PR as well, or should they come with #1839? If the latter, I would remove them from this branch for now.